### ISSUE TITLE
[FR] Allow a node to be bypassed (thereby disabling other nodes connected to it downstream)

### ISSUE THREAD

**Issue by j3ffgray** [breadboard-ai/breadboard#2117]
## Potential Behavior

This would provide the means for future features to be stubbed in to show future business logic or feature exploration.

It could also provide a means of toggling approaches in a single board, so you could do local A/B testing of similar routing while iterating on an idea.

## Possible technical issues

For looper and joiner type nodes, this might be difficult to do automatically, specifically disabling other nodes downstream, and loops and joined nodes may still need to function downstream of a disabled node. Perhaps certain nodes would not automatically be bypassed by an upstream node if they have other nodes connected to them. Unclear what the technical hurdles would be here under the hood.

**Comment by dglazkov** [comment]
Yeah, this would be very hard to implement as a generic functionality.

What if we instead flipped this idea to something like "caching node results"?

Like, if I already have results from a previous run, I could say "just reuse these results" or something like that?

**Comment by j3ffgray** [comment]
Hmmmm, that wouldn't really solve the A/B iteration issue (and possibly bring more confusion). Perhaps nests (nodes + edges of varying complexity) that don't contain an connection to an input or output node could be disabled on run? That way a user could disconnect a piece without having to split it off into its own board, and let multiple approaches live alongside each other in one board.

**Comment by dglazkov** [comment]
This is amazing! Let's see if we can focus on the A/B use case. It's something I do a lot, too. Seems like a really good thing to have.

**Comment by dglazkov** [comment]
@donsbach @paullewis A/B seems important. Maybe not for the MVP, but soon after.

**Comment by paullewis** [comment]
How do you see this behaviour surfacing for the user? I'm not all that familiar with how ComfyUI does it, and I couldn't figure out how to toggle things in the example you linked, @j3ffgray.
